enterprise/tasks.py

```diff
@@ -244,27 +245,29 @@ def send_group_membership_invitation_notification(
 
     braze_trigger_properties['act_by_date'] = act_by_date.strftime('%B %d, %Y')
     pecu_emails = []
-    ecus = []
+    user_id_by_email = {}
     membership_records = enterprise_group_membership_model().objects.filter(uuid__in=membership_uuids)
     for group_membership in membership_records:
         if group_membership.pending_enterprise_customer_user is not None:
             pecu_emails.append(group_membership.pending_enterprise_customer_user.user_email)
         else:
-            ecus.append({
-                'user_email': group_membership.enterprise_customer_user.user_email,
-                'user_id': group_membership.enterprise_customer_user.user_id
-            })
+            user_id_by_email[
+                group_membership.enterprise_customer_user.user_email
+            ] = group_membership.enterprise_customer_user.user_id
     recipients = []
     for pecu_email in pecu_emails:
         recipients.append(braze_client_instance.create_recipient_no_external_id(pecu_email))
     braze_client_instance.create_braze_alias(
         [pecu_emails],
         ENTERPRISE_BRAZE_ALIAS_LABEL,
     )
-    for ecu in ecus:
-        recipients.append(braze_client_instance.create_recipient(
-            user_email=ecu['user_email'],
-            lms_user_id=ecu['user_id']))
+    for user_id_by_email_chunk in batch_dict(user_id_by_email, MAX_NUM_IDENTIFY_USERS_ALIASES):
+        recipients_by_email = braze_client_instance.create_recipients(
+            alias_label=ENTERPRISE_BRAZE_ALIAS_LABEL,
+            user_id_by_email=user_id_by_email_chunk
+        )
+        for _, recipient in recipients_by_email.items():
+            recipients.append(recipient)
```

Comment:
code-golf non-blocker: `recipients.extend(recipients_by_email.values())` will do the same thing.